specification/compute/resource-manager/Microsoft.Compute/2017-03-30/compute.json

```diff
@@ -1368,6 +1368,45 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/instanceView": {
```

Comment:
Instance view is already a property on the virtualMachine. Why do you want to expose it through a separate API? This makes it look like instanceView is a child of virtualMachine which it is not.